starky/Cargo.toml

```diff
@@ -38,13 +38,19 @@ pil_analyzer = {git = "https://github.com/powdr-labs/powdr.git", branch = "main"
 number = {git = "https://github.com/powdr-labs/powdr.git", branch = "main", package = "number"}
 ast = {git = "https://github.com/powdr-labs/powdr.git",   branch = "main",package = "ast"}
 
+profiler_macro = {git = "https://github.com/ChengYueJia/profiler-rs", branch = "main"}
```

Comment:
Can we use features=profiler to determine if we import this?

Reply:
Of course.

Reply:
If so, we can set  up optional = true?